### DISCUSSION TITLE
Multi-line text (a cheat-sheet) on a panel?

### DISCUSSION THREAD

**Discussion by Gnomasz** [lxqt/lxqt#2498] [Q&A]
Hi,
I'd like to place a short help, or a cheat sheet with handy keyboard shortcuts that I want to learn, on a panel on the side of the screen. Is there a way to do this without defining each line of text as a separate Custom Command? (And, optionally, with left-aligned text instead of centered one)?

I have tried `echo -e "Text1 \nText2 \nText3`, but this results in a truncated text. 

I'm on LXQt version 1.3.

**Comment by stefonarch** [comment]
Probably a notes application like feathernotes or similar are an easier way, or a shortcut/icon (quicklaunch) pointing to a text file.
